x-pack/plugins/lens/public/xy_visualization/xy_config_panel/threshold_panel.tsx

```diff
@@ -270,6 +339,43 @@ export const ThresholdPanel = (
           }}
         />
       </EuiFormRow>
+      <EuiFormRow
+        display="columnCompressed"
+        fullWidth
+        label={i18n.translate('xpack.lens.xyChart.thresholdMarker.position', {
+          defaultMessage: 'Icon position',
+        })}
+      >
```

Comment:
In addition to disabling the `EuiButtonGroup`, I believe we also need to disable this `EuiFormRow` component when no icon is selected.

```suggestion
      <EuiFormRow
        display="columnCompressed"
        fullWidth
        isDisabled={currentYConfig?.icon == null || currentYConfig?.icon === 'none'}
        label={i18n.translate('xpack.lens.xyChart.thresholdMarker.position', {
          defaultMessage: 'Icon position',
        })}
      >
```